src/main/java/com/spaceclub/club/controller/ClubController.java

```diff
@@ -0,0 +1,35 @@
+package com.spaceclub.club.controller;
+
+import com.spaceclub.club.controller.dto.CreateClubRequest;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+@RestController
+@RequestMapping("/api/v1")
+public class ClubController {
+
+    @PostMapping("/club")
+    public ResponseEntity<String> createClub(@RequestBody CreateClubRequest request) throws URISyntaxException {
+        return ResponseEntity.created(new URI("https://spaceclub.site/1")).build();
```

Comment:
성원님 이거 .location(URI.create("https://spaceclub.site/1")).build로 수정하면 체크 예외 제거할 수 있어요
```suggestion
        return ResponseEntity.location(URI.create("https://spaceclub.site/1")).build();
```

Reply:
[refactor: checked exception 제거](https://github.com/Space-Club/Backend/pull/7/commits/2c98360c7e10c926074516cd2e714b15e51bf556) 반영